.github/workflows/trivy-all.yml

```
@@ -30,30 +27,61 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-
-      - name: Run Trivy on indy_node:${{ matrix.os_version }}
+​
+      - name: Run Trivy vulnerability scanner for Slack messages on indy_node:${{ matrix.os_version }}
         uses: aquasecurity/trivy-action@master
         with:
-          image-ref: 'ghcr.io/${{ needs.workflow_setup.outputs.repo_owner }}/indy-node-container/indy_node:latest-${{ matrix.os_version }}'
-          format: 'template'
-          template: '@/contrib/sarif.tpl'
-          output: 'trivy-indy-node-${{ matrix.os_version }}.sarif'
+          #image-ref: 'ghcr.io/idunion/indy-node-container/indy_node:latest-ubuntu18'
+          image-ref: 'ghcr.io/zickau/indy_node:latest-${{ matrix.os_version }}'
```

@zickau please run the scan on the images in the HL repo, i.e.

```suggestion
          image-ref: 'ghcr.io/${{ needs.workflow_setup.outputs.repo_owner }}/indy-node-container/indy_node:latest-${{ matrix.os_version }}'
```